service/controller/resource/clusterconfigmap/desired.go

```diff
@@ -46,6 +47,22 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) ([]*cor
 		}
 	}
 
+	var clusterCA string
```

Comment:
Am I being dumb or is this completely unrelated to the basedomain change?

Reply:
This is actually the main change, I just changed `baseDomain` because I found it confusing when working on this. Sorry for the confusion.